app/models/user.rb

```diff
@@ -0,0 +1,5 @@
+class User < ActiveRecord::Base
+  has_many :Post
+  has_many :comment

```

Comment:
- [ ] [MANDATORY] This following associations should like this
```ruby
has_many :posts
has_many :comments
```

Please note that the cases and grammatical numbers are extremely important ad these tells rails what table to look at for this associations.